webbpsf/opds.py

```diff
@@ -2979,7 +2977,7 @@ def random_unstack(ote, radius=1, verbose=False):
 
 def test_OPDbender():
```

Comment:
These two test functions should be deleted. Better tests were long ago added in the tests subdirectory.